content/articles/ordering-standard-certificate.markdown

```diff
@@ -61,6 +61,11 @@ The order is the first step into getting an SSL certificate. It will create an S
 
     ![Purchase a Certificate](/files/dnsimple-certificate-purchase.png)
 
+    <info>
+    The certificate price will be held on your card immediately. Your card will be charged once the certificate is issued.
+    Should anything go wrong during the issuance, funds will be released.
```

Comment:
Can we say "If the issuance fails, funds will be released"? or is that too specific? I'm just a little hesitant that "anything" is a little too broad here.

Reply:
@itsalyse yes, that's perfectly fine.

I will review the other changes for uses of _anything_ and try to find viable alternatives.